tests/__tests__/use-strict.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { } from 'jest';
+import { } from 'node';
+import runJest from '../__helpers__/runJest';
+
+describe('use strict', () => {
+
+  it('should show the error locations for "use strict" violations', () => {
+
+    const result = runJest('../use-strict', ['--no-cache', '-t', 'Invalid Strict']);
```

Comment:
It runs a test that matches the name specified.

Running `jest --help` showed this part:

```
--testNamePattern, -t           Run only tests with a name that matches the
                                  regex pattern.                        [string]

```